.github/workflows/release-binary.yml

```diff
@@ -0,0 +1,32 @@
+name: release-binary
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-binary:
+    name: build-binary
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Setup Node.js version
+        uses: actions/setup-node@master
+        with:
+          node-version: [16.15.0]
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm install
+      - name: Build application
+        run: npm build
+      - name: Bundle
+        run: npm bundle
+      - name: Generate Binary
+        run: npm binary
+      - run: chmod -R ugo+x oidc-*
```

Comment:
You can let away the ugo, because that includes all with user, group and others.
chmod -R +x is doing the same.